src/pynwb/core.py

```diff
@@ -71,7 +71,7 @@ def append(self, arg):
         if isinstance(self.data, list):
             self.data.append(arg)
         elif isinstance(self.data, np.ndarray):
-            self.__data = np.append(self.__data, [arg])
+            self.__data = np.vstack((self.__data, [arg]))
```

Comment:
I think this will not work for 1-dimensional data. 

```python
>>> import numpy as np
>>> data = np.array([1, 2, 3])
>>> np.vstack((data, [[4, 5, 6]]))
array([[1, 2, 3],
       [4, 5, 6]])
>>> np.append(data, [[4, 5, 6]])
array([1, 2, 3, 4, 5, 6])
```

Reply:
oh true ok I'll think of a more general solution